server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -641,14 +646,6 @@ export class ScribeLambda implements IPartitionLambda {
 					this.pendingCheckpointOffset = undefined;
 					this.checkpointCore(pendingScribe, pendingOffset, clearCache);
 				}
-			})
-			.catch((error) => {
```

Comment:
It would be good to still have this `.catch` _after_ the `.then`. Otherwise, if an error is thrown in the `.then`, it will be uncaught. I think this would potentially be distinct from  the first write/delete catch, because it's after the context checkpoint or checkpointCore call possibly failing.

Reply:
Updated the order